src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -410,28 +715,27 @@ mod tests {
             assert!(!ip_sub.matches(&any_cryptography_io));
         }
 
-        // Single SAN, IP range.
```

Comment:
Can these tests changes be split out?

Reply:
https://github.com/pyca/cryptography/pull/9872